apache_bot_blocker/file_reader.py

```diff
@@ -0,0 +1,11 @@
+import pkg_resources
+
+def read_ip_list(resource_name):
+    # Get the resource path
+    data = pkg_resources.resource_string(__name__, resource_name)
+    return data.decode('utf-8').splitlines()
+
+# Access files included in the package
+whitelisted_ips = read_ip_list('whitelist')
```

Comment:
When you put this code outside of a function, it means the code will run at the time when this file is imported. In your case, it is fine. But be careful doing this way. A safer way is to run this line at the time when you need to know the value of whitelisted_ips.